Bio/PDB/qcprot.py

```diff
@@ -108,7 +108,9 @@ def qcp(coords1, coords2, natoms):
         x2 = mxEigenV * mxEigenV
         b = (x2 + C2) * mxEigenV
         a = b + C1
-        delta = (a * mxEigenV + C0) / (2.0 * x2 * mxEigenV + b + a)
+        # Add 1e6 to delta to avoid division by 0 and warnings in certain
+        # cases with coordinate sets that are very similar after transformation.
+        delta = (a * mxEigenV + C0) / ((2.0 * x2 * mxEigenV + b + a) + 1e6)
```

Comment:
Might be a question for @mdehoon as I don't really do much heavy numerical work with Python/Numpy, but there are standard "epsilon" values defined in NumPy - should we use one of those here?

Reply:
@JoaoRodrigues   What should `delta` be if the denominator is zero?

Reply:
https://en.wikipedia.org/wiki/Newton%27s_method#Code

@mdehoon the calculation should stop (line 19 in the wiki example). 

The implementation here (taken directly from the original QCP paper) is sparser with the checks (probably for performance reasons of having two conditionals in the loop). I added the tiny `evalprec` to avoid the check and the runtime exception (warning).

I refactored the code a little and added more comments to make the purpose of the code and the naming of the variables more obvious to the reader.